index.src.html

```diff
@@ -3279,6 +3289,15 @@
 : <dfn attribute for=VideoFrame>[[color space]]</dfn>
 :: The {{VideoColorSpace}} associated with this frame.
 
+: <dfn attribute for=VideoFrame>\[[metadata]]</dfn>
+:: The {{VideoFrameMetadata}} associated with this frame.
+
+<div class='note'>
+{{VideoFrameMetadata}} is designed to be extended by other specifications.
+Specifications that do extend {{VideoFrameMetadata}} must make sure that
+added properties are serializable.
```

Comment:
It might be worth centralising these in the long run but partial dictionaries are pretty nice in terms of modularity.
First user might be FaceDetection related metadata, see https://github.com/w3c/mediacapture-extensions/pull/48.
I think it is worth keeping it in a separate spec until the whole proposal migrates to media capture-main, WebCodecs and HTML if there is a need for rvfc..

I could see additional metadata like the ones exposed by rvfc (rtpTimestamp, receiveTime, captureTime) be candidate as well. These ones could go to WebCodecs directly and we could try to have rvfc metadata dictionary extend WebCodecs VideoFrameMetadata to not duplicate the fields and descriptions.

Also, the fact that properties need to be serializable is exercised when calling metadata(). In theory, implementors would catch such issues if spec authors do not.

Reply:
I agree with Paul that this is problematic. There needs to be a central place where metadata is defined, or at least collected.  One possibility is to define a registry for metadata.  That registry specification would also define the requirements for additions.

Reply:
I am ok with this registry approach, or actual users of metadata to write PRs that would add members to this dictionary.
What changes to this PR are you envisioning?
Something like: 
`By design, all {{VideoFrameMetadata}} properties must be serializable.
Specification editors need to ensure this requirement is met whenever extending {{VideoFrameMetadata}}`.